models/user.js

```diff
@@ -1,5 +1,4 @@
 import connector from "#models/databaseUtil";
-import { logger } from "#util";
```

Comment:
I know logger isnt currently being used, but can you add this line back? We need to make the PRs related to the issue. Anyway, it will be used soon enough.